src/common-principles.md

```diff
@@ -422,6 +422,10 @@ possible. Since the NIfTI standard offers limited support for the various image
 acquisition parameters available in DICOM files, we RECOMMEND that users provide
 additional meta information extracted from DICOM files in a sidecar JSON file
 (with the same filename as the `.nii[.gz]` file, but with a `.json` extension).
+Many of such metadata entries are explicitly formalized in BIDS and could be found listed in the [Glossary](./glossary.md).
+Whenever formalizing a new metadata entry within BIDS, the name SHOULD correspond to the original DICOM Tag whenever the value is taken *as is* without any further harmonization.
+The description of the term in the schema then SHOULD describe such correspondence as "Corresponds to DICOM Tag ID1, ID2 `DICOM Tag Name`.".
+If value is harmonized, the description SHOULD describe such correspondence as "Based on DICOM Tag ID1, ID2 `DICOM Tag Name`.".
```

Comment:
```suggestion
If the name is harmonized (against the previous recommendation), then the description SHOULD describe such correspondence as "Based on DICOM Tag ID1, ID2 `DICOM Tag Name`.".
```

Reply:
it feels that it is two points here:

1. It might indeed be the name and/or the value which is "harmonized". Previous wording was aiming only for `value` harmonization, @sappelhoff , you propose to flip it to `name` here?
    - `name` harmonization:  For the `name`-only (value stays the same) I do not immediately can come up with an example but potentially it could happen.  
    - overall, good point, will think about it a little more.
 2. Overall wording could be improved to make two cases more obvious (need to think more too, I have some dislike to suggested "against ..." formulation)

Reply:
> but potentially it could happen

then let's try to cover both 👍 

I was mostly trying to streamline the language, please feel free to iterate!